### PR TITLE
Support for slices

### LINKED ISSUE #21: Support for slices

### PR DESCRIPTION
**Pull request by YassineE**
Fix #21

### CODE REVIEW

File: config_test.go [hunk 1]
```diff
@@ -386,3 +386,64 @@ func TestCustomTag(t *testing.T) {
...
+			"a",
+			"b",
+			"c",
+		}
+		err := confita.NewLoader(st).Load(context.Background(), &s)
+		require.NoError(t, err)
+		require.EqualValues(t, e, s.Letters)
+	})
+
+	t.Run("Non-empty slice", func(t *testing.T) {
```
Comment (jhchabran) [inline config_test.go]: Is this behaviour what we want? Aren't we introducing a "surprising" feature? For all other fields, we're "replacing" the content when loading stuff. Why would we just append here?

File: config.go [hunk 2]
```diff
@@ -234,6 +234,23 @@ func convert(data string, value *reflect.Value) error {
 		}
 
 		value.SetFloat(f)
+	case k == reflect.Slice:
+		var err error
+		// create a new temporary slice to override the actual Value if it's not empty
+		nv := reflect.MakeSlice(value.Type(), 0, 0)
+		ss := strings.Split(data, ",")
+		for _, s := range ss {
+			// retrieve a new Value v based on the type of the slice
```
Comment (asdine) [inline config.go]: :gb: `retrieve` -> `create`

File: config_test.go [hunk 3]
```diff
@@ -386,3 +386,62 @@ func TestCustomTag(t *testing.T) {
...
+			"e",
+		}
+		err := confita.NewLoader(st).Load(context.Background(), &s)
+		require.NoError(t, err)
+		require.EqualValues(t, e, s.Letters)
+	})
+
+	t.Run("Slice of int", func(t *testing.T) {
+		s := struct {
+			Numbers []int `config:"numbers"`
```
Comment (asdine) [inline config_test.go]: Does it work on slices of pointers ?

Comment (YassineE) [reply config_test.go]: Yep it works. I've added a test. Can you tell me if it is relevant ?

**Pull request merged**